sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/OptimizeInSuite.scala

```diff
@@ -91,21 +91,6 @@ class OptimizeInSuite extends PlanTest {
     comparePlans(optimized, correctAnswer)
   }
 
-  test("OptimizedIn test: In clause not optimized in case filter has attributes") {
```

Comment:
Remove this test since we support convert part of list and new test include this.